target/linux/generic/backport-5.15/344-v5.18-02-phy-marvell-phy-mvebu-a3700-comphy-Add-native-kernel.patch

```diff
@@ -1197,21 +1182,10 @@ Signed-off-by: Marek Behún <kabel@kernel.org>
 +
 +	ret = comphy_lane_reg_poll(lane, COMPHY_PIPE_LANE_STAT1, TXDCLK_PCLK_EN,
 +				   COMPHY_PLL_SLEEP, COMPHY_PLL_TIMEOUT);
-+	if (ret) {
++	if (ret)
 +		dev_err(lane->dev, "Failed to lock PCIE PLL\n");
-+		return ret;
-+	}
 +
-+	return 0;
-+}
-+
-+static void
-+mvebu_a3700_comphy_usb3_power_off(struct mvebu_a3700_comphy_lane *lane)
-+{
-+	/*
-+	 * Currently the USB3 MAC sets the USB3 PHY to low state, so we do not
-+	 * need to power off USB3 PHY again.
-+	 */
```

Comment:
This cut section is from the (now missing) fix:
https://github.com/torvalds/linux/commit/0a6fc70d76bddf98278af2ac000379c82aec8f11

Reply:
so we need to include another patch